zerver/webhooks/beeminder/doc.md

```diff
@@ -0,0 +1,29 @@
+Get Beeminder notifications in zulip whenever you're going to derail from your goal!
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+url structure should be like this:
+
+`{{api_url}}?api_key=BOT'S_API_KEY&email=foo@example.com`
+
+Modify the parameters of the url above where `api_key` is api key of \
+your Zulip bot and `email` is your zulip email.
+
```

Comment:
Okay, so:

* Replace `url` with `URL`.
* Replace `is api key of \` with `is the API key of`
* Capitalize `zulip` in `is your zulip email.`